package.json

```diff
@@ -85,6 +86,10 @@
     "lint:fix": "eslint --fix --ext .js,.vue --ignore-path .gitignore .",
     "storybook": "start-storybook -p 6006",
     "storybook:build": "build-storybook",
-    "test:unit": "vitest run"
+    "test:unit": "vitest run",
+    "prepare": "husky install",
+    "version:patch": "npm version patch",
```

Comment:
Can move these to use something like this `npm --no-git-tag-version version` if you don't want the tag auto created with the command.